mapping/sustainability.yaml

```
@@ -1209,16 +1209,16 @@
 
         For each meeting:
 
-        1. Publish the meeting invite. Add a document, set its `.documentType` to 'consultationMeetingInvite' and its `.url` to the URL at which the meeting invite is available.
+        1. Publish the meeting invite. Add a document, set its `.documentType` to 'consultationMeetingInvitation' and its `.url` to the URL at which the meeting invite is available.
```

Refer to the common operation for adding a project document.

```suggestion
        1. Publish the meeting invite. [Add a project document](../common.md#add-a-project-document) and set its `.documentType` to 'consultationMeetingInvitation'.
```